packages/airnode-node/test/setup/e2e/utils.ts

```diff
@@ -26,6 +26,8 @@ export function buildChainConfig(contracts: Contracts): ChainConfig {
         unit: 'gwei',
       },
       fulfillmentGasLimit: 500_000,
+      // 1 ETH
+      withdrawalRemainder: '1000000000000000000',
```

Comment:
```suggestion
      withdrawalRemainder: utils.parseEther("1").toString(),
```

Reply:
I'm not applying this suggestion as I went with an amount type like @bbenligiray recommended above.